src/labeler.ts

```diff
@@ -15,7 +15,8 @@ export async function run() {
   try {
     const token = core.getInput("repo-token", { required: true });
     const configPath = core.getInput("configuration-path", { required: true });
-    const syncLabels = !!core.getInput("sync-labels", { required: false });
+    const syncLabels =
+      core.getInput("sync-labels", { required: false }) === "true";
```

Comment:
We can use the `getBooleanInput` core function to parse a boolean input:
```suggestion
    const syncLabels = core.getBooleanInput('sync-labels');
```